coralproject/talk
# [next] Client Routing

## What does this PR do?

When a client router is mounted, the client code should handle routing past that point. This PR addresses this issue by using a wildcard to handle all routes under it.

## How do I test this PR?

1. Build via `npm run build:client`
2. Start via `npm run start:development`
4. Visit http://localhost:3000/admin/configure/auth directly, and note that it routes you there after the login.
